assets/css/_search_form.scss

```
@@ -1,95 +1,97 @@
-.c-search-form__row {
+.c-search-form {
   display: flex;
+  flex-flow: column nowrap;
+  gap: 1rem;
+}
 
-  &:not(:last-child) {
-    margin-bottom: 1rem;
-  }
+.c-search-form__search-control {
+  // Make control area match designs
+  height: 2.25rem;
 }
 
-.c-search-form__text {
+.c-search-form__search-input-container {
+  // Layout floating `input-controls` over the `search-input-container`
   position: relative;
-  flex: 1 1 auto;
+
+  // Use bootstrap control as base for now
+  @extend .form-control;
+
+  // fill control defined valid area
+  height: 100%;
+  // Make space for `input-controls`
+  padding-right: calc(2.25rem * 2);
+  // Ignore bootstrap padding due to `input-controls`
+  padding-block: 0;
+
+  border-color: $color-gray-400;
+
+  &:focus,
+  &:focus-within,
+  &:active {
+    border: 1px solid $color-eggplant-500;
+    box-shadow: 0px 0px 0px 2px #aa4ef22b;
+    color: $color-gray-800;
+  }
 }
 
 .c-search-form__input {
+  // Align self within parent boundaries
   width: 100%;
-  height: 2.375rem;
-  box-sizing: border-box;
+  height: 100%;
 
-  border-radius: 4px 0 0 4px;
-  border: none;
+  // Match font size from the Route Picker Search
+  @include font-small;
 
-  padding: 7px 0 7px 13px;
+  // Remove default displays
+  border: none;
+}
 
-  font-size: var(--font-size-s);
-  font-weight: var(--font-weight-reg);
-  color: var(--font-color-on-light-600);
+// #region Search Input Buttons
+.c-search-form__input-controls {
+  position: absolute;
+  // Stick to right side of box, do not intercept mouse events to the text input
+  inset-block: 0;
+  right: 0;
```

**question:** what's the purpose of setting both `inset-block` and `right`? I assume there's some reason but I can't seem to figure it out.

---

I needed to set everything _but_ `left`, here, `inset-block` sets both `top` and `bottom` to `0`.

Alternatively I think I could do `inset: 0` and `left: unset`? Unsure about this though (I thought I tried it).
```suggestion
  // Stick to right side of box, do not intercept mouse events to the text input
  inset: 0;
  left: 0;
```

---

Ah, okay, I was slightly misreading the MDN docs on `inset-block` and that's not a property I was previously familiar with. I think it's fine as-is.

---

On the other hand maybe it's weird to mix `left` / `right` / `top` / `bottom` with properties that depend on text directionality and writing mode stuff, but I guess the practical chance of that being an issue is extremely low.

---

True! I'm gonna rework this

---

[`f812089` (#2128)](https://github.com/mbta/skate/pull/2128/commits/f81208973c7f11e338010fdb826601338d164377)